File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        res = []
        for n in nums:
            idx = abs(n)-1
            if nums[idx] > 0:
                nums[idx] *= -1
        for i in range(0, len(nums)):
            if nums[i] > 0:
                res.append(i+1)
        return res

File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var findDisappearedNumbers = function(nums) {
    for (let i=0; i < nums.length; i++) {
        const abs = Math.abs(nums[i])
        if (nums[abs-1] > 0) {
            nums[abs-1] *= -1
        }
    }
    const res = []
    for (let i=0; i < nums.length; i++) {
        if (nums[i] > 0) {
            res.push(i+1)
        }
    }
    return res
};